.github/workflows/test-sui.yaml

```diff
@@ -36,18 +36,20 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Node.js
-        uses: actions/setup-node@v4
+        uses: axelarnetwork/axelar-cgp-sui/.github/actions/nodejs-ci@main
```

Comment:
weird to use cgp sui's node setup action (which is meant for the repo itself atm). we should use the action from the generic repo once that's setup. So, I'd avoid changing this (adds unnecessary coupling)